data/spec-lanes.json

```diff
@@ -1,28 +1,39 @@
 {
     "$id": "https://a-b-street.github.io/osm2lanes",
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "description": "A representation of lanes",
+    "description": "Lanes from left to right.",
     "type": "array",
     "items": {
         "type": "object",
+        "description": "A lane. Can either be for travelling, parking, or any visible separation between them.",
         "required": [
-            "type",
-            "direction"
+            "type"
         ],
         "properties": {
             "type": {
                 "type": "string",
                 "description": "The type of the lane.",
-                "enum": [
-                    "buffer",
-                    "bus_lane",
-                    "construction",
-                    "cycleway",
-                    "parking_lane",
-                    "shared_left_turn",
-                    "shoulder",
-                    "sidewalk",
-                    "travel_lane"
+                "anyOf": [
+                    {
+                        "const": "travel",
```

Comment:
So the big change is to shift from existing "categories" of common lane types like cycleways and shared left turn lanes, to a general travel lane that decomposes concerns. `designated` seems to be the access restrictions -- so a "bus lane" also usable by taxis and cyclists can be indicated directly. Is that the idea?

Reply:
this is to help clients differentiate lanes by the common use cases, e.g. the options in streetmix